### ISSUE TITLE
connect deprecated methodOverride since express update

### ISSUE THREAD

**Issue by Kikobeats** [balderdashy/sails#2316]
I see this warning since the [last express update commit](https://github.com/balderdashy/sails/commit/d40a60bb5fd41d7e29c80fb44e99edb41fb34773):
![](http://i.imgur.com/xqg6I1U.png)

**Comment by CWyrtzen** [comment]
Is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @Kikobeats.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
